### ISSUE TITLE
cannot visualize as pointclound 

### ISSUE THREAD

**Issue by ZoyeJay** [lmb-freiburg/demon#48]
I runed the example.py, and it can show the depth map, but cannot visualize the pointclound. I installed the vtk on python3. and no error on "import vtk".
![error](https://user-images.githubusercontent.com/30227272/42980962-46fcbef8-8c0c-11e8-89e5-a5bf0593accd.PNG)
So please help me solve this error. Its about Cython? OR other? THANKS A LOT~

**Comment by benjaminum** [comment]
Yes, it seems to be about Cython and numpy.
Did you try to update numpy or Cython?

**Comment by ZoyeJay** [comment]
yes. The Cython version is 0.28.4 and the numpy version is 1.15.0. They are both the newest, but it still have the same error.
